ci/goreleaser/goreleaser-el7.yml

```diff
@@ -14,6 +14,9 @@ builds:
       - -X github.com/TykTechnologies/tyk/gateway.VERSION={{.Version}} -X github.com/TykTechnologies/tyk/gateway.Commit={{.FullCommit}} -X github.com/TykTechnologies/tyk/gateway.buildDate={{.Date}} -X github.com/TykTechnologies/tyk/gateway.builtBy=goreleaser
     goos:
       - linux
+    env:
+      - CGO_ENABLED=1
+      - GO111MODULE=off
```

Comment:
This change needs to be in templates right? Better commit it right now.

Reply:
Updated in https://github.com/TykTechnologies/gromit/pull/201 and synced from it in https://github.com/TykTechnologies/tyk/pull/5428/commits/9ef5d1f769bbc248a589d77a7413b41e59dfcb89